app/assets/javascripts/file_field.js

```diff
@@ -0,0 +1,21 @@
+function FileField(elementIdToBeAppendedIn) {
+  this.$elementToBeAppendedIn = $('#' + elementIdToBeAppendedIn);
+}
+
+FileField.prototype.createFileFieldHtml = function() {
+  var uniqueString = new Date().toString();
```

Comment:
Rename to `fieldTimestamp`. uniqueString doesn't suggest the purpose of the variable.